x_xy/render.py

```diff
@@ -366,7 +364,11 @@ def animate(
 
     if verbose:
         print(f"DONE. Converting frames to {path} (this might take a while..)")
-    imageio.mimsave(path, frames, format=fmt, fps=fps)
+
+    if len(frames) == 1 and (fmt == "jpg" or fmt == "png"):
```

Comment:
Then, i would prefer if we would require that `xs` is of type `Transform | list[Transform]` which then nicely indicates that this function can also make images. 
Then, `Transform.ndim() == 1` in this function so to speak.

Reply:
I would also suggest changing the `fmt` parameter to default to `None`, since currently the function has to be called as e.g. `render.animate("figures/example.png", sys, x, fmt="png")` and can usually be easily inferred from the file extension.

Alternatively we could just remove it entirely, as it honestly doesn't seem that useful (as, again, it can usually just be inferred from the file extension).

Reply:
Done, can now render mp4, png, and jpg. For images if a sequence of Transforms is given, renders the first timestep.

Some other formats can probably be added by simply adding them to the respective image/video format lists, but should be tested for unexpected pitfalls (e.g. for jpg the alpha channel has to be removed).